ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/castor/algorithm/CastorFullOptimization.java

```diff
@@ -80,13 +81,14 @@ public CompletableFuture<RaoResult> run() {
             toolProvider);
 
         PrePerimeterResult initialOutput;
-        initialOutput = prePerimeterSensitivityAnalysis.runInitialSensitivityAnalysis(network, crac);
+        initialOutput = prePerimeterSensitivityAnalysis.runInitialSensitivityAnalysis(network, crac, Set.of());
```

Comment:
instead of passing an empty set, create another method without the last argument